projects/notebooks/visualizations/rhods-notebooks-performance/analyze/__init__.py

```diff
@@ -1,13 +1,39 @@
-import matrix_benchmarking.common as common
+import json
 import logging
+import numpy as np
+
+from copy import deepcopy
+
+import matrix_benchmarking.common as common
+import matrix_benchmarking.regression as regression
 
 def run():
-    logging.info(f"Received {common.Matrix.count_records()} new entries")
-    for entry in common.Matrix.all_records(): pass
 
     logging.info(f"Received {common.LTS_Matrix.count_records()} historic LTS entries")
-    for lts_entry in common.LTS_Matrix.all_records(): pass
+    lts_entries_raw = list(common.LTS_Matrix.all_records())
```

Comment:
you don't really need the `list()` here. Without it,  `lst_entries_row` is a generator, that is processed (only once) at the line below
removing `list()` saves memory and processing time, especially if the list gets large

Reply:
Fair point!